src/System.Net.HttpListener/src/System/Net/HttpListenerPrefixCollection.cs

```diff
@@ -39,6 +39,11 @@ internal HttpListenerPrefixCollection(HttpListener listener)
         public void CopyTo(Array array, int offset)
         {
             _httpListener.CheckDisposed();
+            // Preserve compatability with full .NET Framework.
+            if (array == null)
+            {
+                throw new NullReferenceException();
```

Comment:
@priya91 @davidsh @stephentoub what are your thoughts on fixing this? It's a bug in .net core, .net desktop but not in the managed implementation. This aligns all 3 to throw an NRE